samples/bootloader/prj.conf

```diff
@@ -15,15 +15,17 @@ CONFIG_SYS_CLOCK_EXISTS=n
 CONFIG_NRF_RTC_TIMER=n
 CONFIG_CLOCK_CONTROL=n
 CONFIG_FPROTECT=y
+CONFIG_FW_INFO=y
```

Comment:
Why does this need to be set explicitly?

Reply:
`FW_INFO` is no longer `selected` in the bootloader so this makes sense now :)